roles/mongodb/tasks/main.yml

```diff
@@ -0,0 +1,40 @@
+---
+- name: install mongodb packages
+  apt: pkg={{ item }} state=installed
+  with_items:
+    - mongodb-org-server={{ mongodb.version }}
+    - mongodb-org-shell={{ mongodb.version }}
```

Comment:
Will the package upgrade cause a service restart, or should this task also trigger a restart of the service (when a mongo version upgrade happens)?